src/dra/5298C.c

```diff
@@ -459,7 +459,7 @@ void DrawMenuSprite(
     MenuContext* context, s32 x, s32 y, s32 width, s32 height, s32 u, s32 v,
     s32 clut, s32 tpage, s32 arg9, s32 colorIntensity, s32 argB) {
     u32* ot = g_CurrentBuffer->ot;
-    POLY_GT4* poly = &g_CurrentBuffer->polyGT4[g_GpuUsage.gt4];
+    Primitive* poly = &g_CurrentBuffer->polyGT4[g_GpuUsage.gt4];
```

Comment:
We have to change it back to `POLY_GT4*`. I know we are still passing it to `SetTexturedPrimRect` but it is fine as the order of fields is the same.